exercises/practice/collatz-conjecture/.meta/config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "blurb": "Calculate the number of steps to reach 1 using the Collatz conjecture.",
+  "authors": [
+    "Kryo"
+  ],
+  "contributors": [],
+  "files": {
+    "solution": [
+      "collatzConjecture.ps1"
+    ],
+    "test": [
+      "collatzConjecture.tests.ps1"
+    ],
+    "example": [
+      ".meta/collatzConjecture.example.ps1"
+    ]
```

Comment:
All the existing files use PascalCase for their files. That's also how other .NET platform languages define their file name, so I think it makes sense to use that here too. Unless I'm missing a specific convention?

Reply:
No that is right. I will correct this and re-push..